jdaviz/core/loaders/importers/spectrum2d/spectrum2d.py

```diff
@@ -106,10 +113,12 @@ def output(self):
         data = hdu.data
         header = hdu.header
         metadata = standardize_metadata(header)
-        metadata[PRIHDR_KEY] = standardize_metadata(hdulist[0].header)
+        if hdu.name != 'PRIMARY' and 'PRIMARY' in hdulist:
+            metadata[PRIHDR_KEY] = standardize_metadata(hdulist[0].header)
         wcs = WCS(header, hdulist)
-        if self.transpose:
+        if data.shape[0] > data.shape[1]:
```

Comment:
i know we talked about this but, now that i'm thinking about it, couldn't there be a reasonable case where you don't want to automatically transpose the data if your spectral axis is shorter? could this be a warning with the ability to switch it off somewhere? it also might be good to add a test case for this situation to make sure everything is transposed

Reply:
what scenario?  If so, we can always add a dropdown that defaults to "auto" or something like that... but I don't want to overcomplicate things if we don't know of a case where we would even want to override the automated logic

Reply:
ok - added a snackbar and we can always add more advanced functionality later if there is a request for it.